multiaccounts/accounts/database.go

```diff
@@ -35,6 +36,47 @@ type Account struct {
 	Color     string         `json:"color"`
 }
 
+type Settings struct {
+	// required
+	Address                types.Address    `json:"address"`
+	ChaosMode              bool             `json:"chaos-mode?,omitempty"`
+	Currency               string           `json:"currency,omitempty"`
+	CurrentNetwork         string           `json:"networks/current-network"`
+	CustomBootnodes        *json.RawMessage `json:"custom-boot-nodes,omitempty"`
+	CustomBootnodesEnabled *json.RawMessage `json:"custom-boot-nodes-enabled,omitempty"`
+	DappsAddress           types.Address    `json:"dapps-address"`
+	EIP1581Address         types.Address    `json:"eip1581-address"`
+	Fleet                  *string          `json:"fleet,omitempty"`
```

Comment:
Any reason it is a pointer while other string fields are not?

Reply:
Because it is nullable others are not. The whole things works well in status-react. I only have an issue with the e2e test